.github/workflows/scala3-build.yml

```diff
@@ -22,13 +21,11 @@ jobs:
           - akka-testkit/test akka-actor-tests/test
           - akka-actor-testkit-typed/test akka-actor-typed-tests/test
           - akka-bench-jmh/test
-          - akka-cluster/Test/compile akka-cluster-tools/test akka-cluster-typed/test akka-distributed-data/test akka-cluster-metrics/Test/compile akka-cluster-sharding/Test/compile
-          - akka-coordination/test
-          - akka-discovery/test
+          - akka-cluster/Test/compile akka-cluster-tools/test akka-cluster-typed/test akka-distributed-data/test akka-cluster-metrics/Test/compile akka-cluster-sharding/Test/compile akka-cluster-sharding-typed/Test/compile
```

Comment:
oh nice when `akka-cluster-sharding-typed/Test/compile` works already

Reply:
eh, yeah I noticed afterwards that I might have been too updating this - but great if it works